spec/classroom_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './../classroom'
+
+describe 'Classroom class' do
+  context 'This tests for the classroom class' do
+    it 'should test the books' do
+      classroom = Classroom.new('label') do
+        expect(classroom.label).to eq 'label'
+      end
+    end
+  end
+end
```

Comment:
Same thing for `classroom.rb`, since it has the `add_student` method. Kindly test the method as well using the guidelines I explained in the above comments.